### ISSUE TITLE
email all participants markdown to html broken

### ISSUE THREAD

**Issue by lbeziaud** [codalab/codabench#930]
Hi,

The formatting of emails sent through the "email all participants" functionality is broken. HTML tags are escaped. For example, sending:
```markdown
**bold** [link](https://codabench.org)
```
results in:
```
--===============0869688155652953062==
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit

<p>&lt;strong&gt;bold&lt;/strong&gt; &lt;a href="https://codabench.org"&gt;link&lt;/a&gt;</p>

--===============0869688155652953062==
Content-Type: text/html; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit

<p>&lt;strong&gt;bold&lt;/strong&gt; &lt;a href="https://codabench.org"&gt;link&lt;/a&gt;</p>
--===============0869688155652953062==--
```
which renders as:
```
<strong>bold</strong> <a href="https://codabench.org">link</a>
```
rather than
> **bold** [link](https://codabench.org)

**Comment by ihsaan-ullah** [comment]
<img width="635" alt="Screenshot 2023-06-12 at 1 53 21 AM" src="https://github.com/codalab/codabench/assets/13259262/262dec65-6536-4b55-9694-13b0f17a771e">

**Comment by ihsaan-ullah** [comment]
> Thank you for reporting this issue. Is it related to this change or is it a separate part of the code?
> 
> * [html is now interpreted in compettion pages #878](https://github.com/codalab/codabench/pull/878)

This seems to be a separate issue. It looks like it is never tested with HTML
This function receives raw html and while sending,  the html is not properly parsed (needs investigation)
https://github.com/codalab/codabench/blob/e7de84f703a5a56636800f0545be538795560194/src/utils/email.py#LL42C5-L42C32

**Comment by Didayolo** [comment]
> How can this be tested locally?

Maybe you can try "to send an email to participants" locally and then find the raw body of the email inside the Django logs using `docker-compose logs django`.

**Comment by lbeziaud** [comment]
You might not need that. The form sends the data as HTML to the API, rather than in plain markdown as requested by https://github.com/codalab/codabench/blob/b728d062712e09e462f8cf422ce90a8166c79244/src/apps/competitions/tasks.py#L743C33

**Comment by ihsaan-ullah** [comment]
Solved by #1340
